wagtail/snippets/views/snippets.py

```diff
@@ -743,6 +746,14 @@ def revisions_compare(self):
             history_url_name=self.get_url_name("history"),
         )
 
+    @property
+    def preview_on_add(self):
```

Comment:
Could we suffix all view names with ``_view`` for consistency with the original views?

Sorry, looks like I missed this one with ``revisions_revert`` and ``revisions_compare``

Reply:
Ahh yes, I missed that as well. Thanks!